tests/__init__.py

```diff
@@ -1,4 +1,261 @@
+import collections
+import copy
+import os
+
+import awscli
+from awscli.clidriver import create_clidriver
 from awscli.compat import collections_abc
+from awscli.testutils import capture_output
+
+import mock
+import botocore.awsrequest
+import botocore.loaders
+import botocore.model
+import botocore.serialize
+import botocore.validate
+
+
+# A shared loader to use for classes in this module. This allows us to
+# load models outside of influence of a session and take advantage of
+# caching to speed up tests.
+_LOADER = botocore.loaders.Loader()
+
+
+class CLIRunner(object):
+    """Runs CLI commands in a stubbed environment"""
+    def __init__(self, env=None):
+        if env is None:
+            env = self._get_default_env()
+        self.env = env
+        self._session_stubber = SessionStubber()
```

Comment:
Can we inject this? Right now this is hardcoded into CLIRunner, and the response classes are hardcoded into this stubber. So we could make it easier to inject your own abstractions if we make all of this injectable.